app/code/VConnect/OrderVolume/etc/events.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
+    <event name="sales_model_service_quote_submit_success">
```

Comment:
Are you sure we need to do it in this event?
This case you need to:
1. Re save an order after it is placed
 OR
2. Make direct sql queries to a database in order to add a volume

I personally prefer to use an event before order placement (replace `success` to `before` in the event name), assign your data to order object and it will be saved into database if everything if written fine.